lib/thumbs/pull_request_worker.rb

```diff
@@ -920,6 +920,10 @@ def comment_code_approvals
       code_reviews
     end
 
+    def remove_build_dir
+      FileUtils.mv(@build_dir, "#{@build_dir}.#{DateTime.now.strftime("%s")}")
```

Comment:
Heh, its already in tmp :) (by default, this way it'll move it relative to the build dir.